src/hooks.ts

```diff
@@ -144,6 +145,11 @@ export const useDrag = ({ onStart, onMove, onEnd, containerRef }: UseDragProps)
         // we don't want to handle clicks other than left ones
         return
       }
+
+      if (knobs.length && !knobs.find(knob => knob.contains(e.target))) {
```

Comment:
As you mentioned, this should be done for the onTouchStart event as well.

Reply:
Done :)